web/impact/impact/v1/helpers/matching_industry_criterion_helper.py

```diff
@@ -12,13 +12,20 @@
 class MatchingIndustryCriterionHelper(MatchingCriterionHelper):
     application_field = "startup__primary_industry__id"
     judge_field = "expertprofile__primary_industry__name"
+    industries = None
 
     def __init__(self, subject):
         super().__init__(subject)
         self._top_level_id_cache = None
 
+    @classmethod
+    def _industry_map(cls):
+        if cls.industries is None:
+            cls.industries = cls.cache_instances_by_name(Industry)
+        return cls.industries
+
     def app_ids_for_feedbacks(self, feedbacks, option_name, applications):
-        target = Industry.objects.filter(name=option_name).first()
+        target = self.__class__._industry_map()[option_name]
```

Comment:
use MatchingIndustryCriterionHelper. _industry_map() instead

Reply:
Sadly, that leads to a ridiculously long line

Reply:
you should still be able to get this pass pep8